src/dynarec/rv64/dynarec_rv64_660f.c

```diff
@@ -419,6 +418,41 @@ uintptr_t dynarec64_660F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
                         FCVTDL(v0, x5, RD_RTZ);
                     }
                     break;
+                case 0x09:
+                    INST_NAME("ROUNDPD Gx, Ex, Ib");
+                    nextop = F8;
+                    GETGX(x1);
+                    GETEX(x2, 1);
+                    u8 = F8;
+                    d0 = fpu_get_scratch(dyn);
+                    d1 = fpu_get_scratch(dyn);
+                    v1 = fpu_get_scratch(dyn);
+                    MOV64x(x3, 1ULL << __DBL_MANT_DIG__);
+                    FCVTDL(d1, x3, RD_RTZ);
+                    for (int i=0; i<2; ++i) {
+                        FLD(d0, wback, fixedaddress+8*i);
+                        FEQD(x4, d0, d0);
+                        BNEZ_MARK(x4);
+                        B_MARK3_nocond;
+                        MARK; // d0 is not nan
```

Comment:
@ptitSeb  Just to be sure, is `MARK` macros going to work inside a for loop?

Reply:
Ah no, it will not. It will just get the MARK for the last iteration

Reply:
It's computed and applied in different passes, right?

Reply:
Yes. It's computed on pass2 and applied on pass3.

Reply:
Ok, I'll fix this in the upcoming PR.